src/windows/keycodes.rs

```diff
@@ -1,6 +1,18 @@
 use crate::rdev::Key;
 use std::convert::TryInto;
 use winapi::shared::minwindef::WORD;
+use winapi::um::winuser::{MapVirtualKeyA, MAPVK_VK_TO_VSC};
+
+pub fn scan_from_code(code: WORD) -> Option<WORD> {
+    let scan = unsafe { MapVirtualKeyA(code as u32, MAPVK_VK_TO_VSC) } as WORD;
```

Comment:
I think this should be either `MapVirtualKey` or `MapVirtualKeyW` instead ([source](https://learn.microsoft.com/en-us/windows/win32/intl/conventions-for-function-prototypes))